src/pages/BeginAccountTransfer/BeginAccountTransfer.tsx

```diff
@@ -42,7 +44,7 @@ export default function BeginAccountTransfer() {
   const { active, signer, account } = useWallet();
   const { chainId } = useChainId();
   const [, setPendingTxns] = usePendingTxns();
-  const [receiver, setReceiver] = useState("");
+  const [receiver, setReceiver] = useState("0x5f8Cf0d45487301665d356c626fE099B43628538");
```

Comment:
Is it a test value?

Reply:
I thought someone could send some tokens to my test acc :D
yeah, forgot to remove it